tea/scenes/apply.go

```diff
@@ -149,13 +150,13 @@ func (m apply) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 	switch msg := msg.(type) {
 	case tea.KeyMsg:
 		switch keypress := msg.String(); keypress {
-		case KeyControlC:
+		case keys.KeyControlC:
 			return m, tea.Quit
-		case KeyEscape:
+		case keys.KeyEscape:
 			m.cancelled = true
 			m.cancelChannel <- true
 			return m, nil
-		case KeyEnter:
+		case keys.KeyEnter:
```

Comment:
bubbletea provides constants for key codes[^1]. Simply switch on `msg.Type`.

[^1]: https://pkg.go.dev/github.com/charmbracelet/bubbletea#KeyType